base/abstractarray.jl

```diff
@@ -209,7 +209,44 @@ String
 """
 valtype(A::Type{<:AbstractArray}) = eltype(A)
 
+"""
+    prevind(::AbstractArray, i::Integer)
+
+Return the index before `i`, equivalent to `i - 1`.
```

Comment:
Maybe add a comment (here and below) for why this method exists:
```suggestion
Return the index before `i`, equivalent to `i - 1`.   (This function can be useful
for generic code that operates on both arrays and strings, since in the latter
case the indices may not be consecutive.)
```

Reply:
Agreed, added in 3422e96622.